components/navigation/BlogPostItem.tsx

```diff
@@ -77,47 +77,46 @@ export default forwardRef(function BlogPostItem(
                   </span>
                 </Paragraph>
-                <Link href={post.slug}>
-                  <span className='block'>
-                    <Heading level={HeadingLevel.h5} typeStyle={HeadingTypeStyle.smSemibold} className='mt-2'>
-                      {post.title}
-                    </Heading>
-                    <Paragraph typeStyle={ParagraphTypeStyle.sm} className='mt-3'>
-                      <TextTruncate element='span' line={4} text={post.excerpt} />
-                    </Paragraph>
-                  </span>
-                </Link>
+                <span className='block'>
+                  <Heading level={HeadingLevel.h5} typeStyle={HeadingTypeStyle.smSemibold} className='mt-2'>
+                    {post.title}
+                  </Heading>
+                  <Paragraph typeStyle={ParagraphTypeStyle.sm} className='mt-3'>
+                    <TextTruncate element='span' line={4} text={post.excerpt} />
+                  </Paragraph>
+                </span>
               </div>
               <div className='mt-6 flex items-center'>
                 <div className='relative shrink-0'>
                   <AuthorAvatars authors={post.authors} />
                 </div>
                 <div className='ml-3'>
                   <Heading level={HeadingLevel.h3} typeStyle={HeadingTypeStyle.xsSemibold} textColor='text-gray-900'>
-                    <span className='hover:underline'>
+                    <span>
                       {post.authors
                         .map((author, index) =>
                           author.link ? (
-                            <a
+                            <button
                               key={index}
                               data-alt={author.name}
-                              href={author.link}
+                              className='cursor-pointer border-none bg-inherit p-0 hover:underline'
                               onClick={(e) => {
-                                e.stopPropagation();
+                                e.preventDefault();
+
+                                // Handle the click event, e.g., navigate to author.link
+                                window.open(author.link, '_blank');
                               }}
-                              target='_blank'
-                              rel='noreferrer'
                             >
                               {author.name}
-                            </a>
+                            </button>
                           ) : (
                             author.name
                           )
                         )
-                        .reduce((prev, curr) => (
-                          <>
+                        .reduce((prev, curr, index) => (
+                          <React.Fragment key={`author-${index}`}>
                             {prev} & {curr}
-                          </>
+                          </React.Fragment>
```

Comment:
button tag can't be replaced with a tag because it already contains Link component in the parent. Hence, adding a <a> tag will lead to hydration issue.